cmd/nodeup/main.go

```diff
@@ -64,17 +68,47 @@ func main() {
 	retries := flagRetries
 
 	for {
-		cmd := &nodeup.NodeUpCommand{
-			ConfigLocation: flagConf,
-			Target:         target,
-			CacheDir:       flagCacheDir,
-			FSRoot:         flagRootFS,
-			ModelDir:       models.NewAssetPath("nodeup"),
-		}
-		err := cmd.Run(os.Stdout)
-		if err == nil {
-			fmt.Printf("success")
-			os.Exit(0)
+		var err error
+		if install {
+			var command []string
+			for i := 0; i < len(os.Args); i++ {
```

Comment:
This is creating a systemd process with the same args as we were called with.  I'll add a comment